rules/windows/file_event/file_event_win_cscript_wscript_dropper.yml

```diff
@@ -0,0 +1,36 @@
+title: WScript or CScript Dropper
+id: 002bdb95-0cf1-46a6-9e08-d38c128a6127
+status: experimental
+description: Detects a file ending in jse, vbe, js, vba, vbs written by cscript.exe or wscript.exe
+related: 
+    - id: cea72823-df4d-4567-950c-0b579eaf0846
+      type: derived
+references:
+    - WScript or CScript Dropper (cea72823-df4d-4567-950c-0b579eaf0846)
+author: Tim Shelton
+date: 2022/01/10
+modified: 2022/01/10
+logsource:
+    category: file_event
+    product: windows
+detection:
+    selection1:
+        Image|endswith:
+            - '\wscript.exe'
+            - '\cscript.exe'
+        TargetFilename|startswith: 
+            - 'C:\Users\'
+            - 'C:\ProgramData'
+        TargetFilename|endswith: 
+            - '.jse'
+            - '.vbe'
+            - '.js'
+            - '.vba'
+            - '.vbs'
+    condition: 1 of selection*
```

Comment:
As there is only 1 selection ( Image AND TargetFilename AND TargetFilename) simply `condition:  selection1`

Reply:
Agreed, only if targetfilename is the only field and will be the only field for the foreseeable future.

Reply:
this has been completed